### DISCUSSION TITLE
Consider enabling GitHub Discussions for this repository

### DISCUSSION THREAD

**Discussion by meator** [crazy-complete/crazy-complete#11] [General]
I would like to share my progress with integrating crazy-complete into j4-dmenu-desktop and discuss crazy-complete, its features and its goals in general. I have been using issues so far, but I don't want to create issues about this, because the things I want to talk about aren't "issues". Using issues for this stuff is inappropriate. Please enable [GitHub Discussions](https://docs.github.com/en/discussions) for this repo. I don't want to spam issues.

**Comment by braph** [comment]
Hello, I'm having trouble enabling discussions for this repository (they seem to be enabled already), so I enabled discussions for this organization with this repository as the target. Are you able to create discussions now?

**Comment by meator** [comment]
Thanks!

**Comment by meator** [comment]
The discussion page is tied to the `crazy-complete` GitHub organisation instead of the repository. I didn't know this was possible. This is quite unusual. It should be possible to just go into repository settings and tick support for Discussions to enable them.
